linkedin.py

```diff
@@ -454,11 +454,19 @@ def handleMultiplePages(self, jobPage, jobProperties: models.Job, jobCounter: mo
             jobCounter = self.cannotApply(jobPage, jobProperties, jobCounter)
             return jobCounter
         
-        percentageElement = self.driver.find_element(By.XPATH, constants.multiplePagePercentageXPATH)
-        comPercentage = percentageElement.get_attribute("value")
-        percentage = int(comPercentage)
-        applyPages = math.ceil(100 / percentage) - 2
```

Comment:
Instead of moving this to the try/catch block, you can use utils.exists(...)

Reply:
Removed the `try-catch` block, replaced it with a `exists` check.